tech.json

```diff
@@ -1313,8 +1244,21 @@
         {
           "name": "canCeilingClip",
           "requires": [],
-          "note": "Basic variant of jumping into a two block high space and performing the actions needed to clip up through a one-tile-thick ceiling.",
-          "devNote": "This often uses enemies, so is required for enemy clip enemies that are extensions of canUseEnemies."
+          "note": [
+            "Basic variant of jumping into a two block high space and performing the actions needed to clip up through a one-tile-thick ceiling.",
+            "Common examples include positioning snails or frozen enemies close enough to the ceiling in order to morph on the enemy, X-Ray Stand Up, and jump through."
+          ],
+          "devNote": "This often uses enemies, so is required for enemy clip enemies that are extensions of canUseEnemies.",
+          "extensionTechs": [
+            {
+              "name": "canPreciseCeilingClip",
+              "requires": [
+                "canCeilingClip",
+                "canUseEnemies"
+              ],
+              "note": "Setting up a an enemy positioning to perform a very precise ceiling clip. The enemy positioning will require a 2-3 pixel range, so the clip can work without the leniency allowed by an X-Ray standup."
```

Comment:
```suggestion
              "note": "Setting up an enemy positioning to perform a very precise ceiling clip. The enemy positioning will require a 2 pixel range, so the clip can work without the leniency allowed by an X-Ray standup."
```

Reply:
Hijump isn't being modelled for the 3rd pixel

Reply:
I got 2-3 pixels without HJB and 3 pixels with